modules/serverless-rn-1-36-0.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-36-0_{context}"]
+= Red Hat {ServerlessProductName} 1.36
+
+{ServerlessProductName} 1.36 is now available. New features, updates, fixed issues, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="new-features-1-36-0_{context}"]
+== New features
+
+[id="new-features-eventing-1-36-0_{context}"]
+=== {ServerlessProductName} Eventing
+
+* {ServerlessProductName} now uses Knative Eventing 1.16.
+
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.16.
+
+* `IntegrationSource` and `IntegrationSink` are now available as a Technology Preview. These are Knative Eventing custom resources that support selected Kamelets from the Apache Camel project. Kamelets enables you to connect to third-party systems for improved connectivity, acting as either sources (event producers) or sinks (event consumers).
+
+* Knative Eventing can now automatically discover and register EventTypes based on the structure of incoming events. This feature simplifies the configuration and management of EventTypes, reducing the need for manual definitions. This feature is available as a Technology Preview.
+
+* {ServerlessProductName} Eventing introduces `EventTransform`, a new API resource that you can use to declaratively transform JSON events without writing custom code. With `EventTransform`, you can modify attributes, extract or reshape data, and streamline event flows across systems. Common use cases include event enrichment, format conversion, and request-response transformation. `EventTransform` integrates seamlessly with Knative sources, triggers, and brokers, enhancing interoperability in event-driven architectures. This feature is now available as a Technology Preview.
```

Comment:
I see in the Knative docs that they use EventTransform without mark-up or _object_. I don't think we should, and should follow our standard 
```suggestion
* {ServerlessProductName} Eventing introduces `EventTransform` objects, a new API resource that you can use to declaratively transform JSON events without writing custom code. With `EventTransform` objects, you can modify attributes, extract or reshape data, and streamline event flows across systems. Common use cases include event enrichment, format conversion, and request-response transformation. Event transformations integrate seamlessly with Knative sources, triggers, and brokers, enhancing interoperability in event-driven architectures. This feature is now available as a Technology Preview.
```